### PR TITLE
Image & input 

### PR DESCRIPTION
**Pull request by Bibin-Antony**
minor changes made

### CODE REVIEW

File: Container/CSS/image.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import url(/Container/CSS/style.css);
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
```
Comment (rohanmathur91) [inline Container/CSS/image.css]: Put this reset in the base CSS file, so need to write for each component.

File: Container/CSS/image.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.responsive-img{
+    width: 100%;
+  height: auto;
+}
+
+.rounded{
+    border-radius: 50%;
+    width: 20rem;
+    height: 20rem;
+  height: auto;
```
Comment (rohanmathur91) [inline Container/CSS/image.css]: Indentation is not proper.

File: Container/CSS/input.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
+@import url(/Container/CSS/style.css);
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
```
Comment (rohanmathur91) [inline Container/CSS/input.css]: same as above.

**Review by rohanmathur91 — COMMENTED** [review]
Added few comments have a look.

**Comment by Bibin-Antony** [comment]
https://github.com/Bibin-Antony/Component-Library-/pull/7

**Comment by Bibin-Antony** [comment]
image

**Comment by Bibin-Antony** [comment]
image

**Review by biswanath-tewari — APPROVED** [review]
looks all good.

File: Container/CSS/input.css (hunk 3 above)
Comment (Bibin-Antony) [reply Container/CSS/input.css]: done

**Comment by Bibin-Antony** [comment]
changes made as necessary.

**Pull request merged**